getting-started/quickstart.md

```diff
@@ -107,7 +107,7 @@ You can see your application **logs** in the dashboard to **monitor the deployme
    <p>If the remote asks you for a password right after a git push attempt, this may be due to a SSH Key misconfiguration.
    <br>Add your SSH key to your profile here:
    <a href="https://console.clever-cloud.com/users/me/ssh-keys">https://console.clever-cloud.com/users/me/ssh-keys</a></p>
-   <p>The full tutorial about adding SSH key is here: <a href="/doc/admin-console/ssh-keys/">Adding SSH keys</a> </p>
+   <p>The full tutorial about adding SSH key is here: <a href="/admin-console/ssh-keys/">Adding SSH keys</a> </p>
```

Comment:
It seems like the whole path also changed. While your commit fixes it, the user still gets redirected to the right place. I think it would be better to update the path altogether.

```suggestion
   <p>The full tutorial about adding SSH key is here: <a href="/account/ssh-keys-managment/">Adding SSH keys</a> </p>
```

Reply:
I do agree! I have hesitated to do that and was thinking it was for any reason wanted.